zboxcore/sdk/chunked_upload.go

```diff
@@ -594,12 +597,27 @@ func (su *ChunkedUpload) processUpload(chunkStartIndex, chunkEndIndex int,
 			defer wg.Done()
 			err = b.sendUploadRequest(ctx, su, chunkEndIndex, isFinal, encryptedKey, body, formData, pos)
 			if err != nil {
+				select {
+					case wgErrors <- err:
```

Comment:
what is the reason for this select? is it the same as next `wgErrors <- err` write?

Reply:
@Manish-210 may forget to remove the `wgErrors <- err`.

Reply:
Sorry I forgot to remove it on line 604. Thanks @peterlimg & @dabasov for pointing it out.